services/qs-spark-service/src/main/resources/application-sat.properties

```diff
@@ -26,6 +26,8 @@ qs.aws.config.buckets=${CONFIG_BUCKETS_NAME}
 qs.client.name=${CLIENT_NAME}
 qs.aws.quantumics.bucket.region=${QUANTUMICS_BUCKETS_REGION}
 qs.aws.natwest.bucket.region=${NATWEST_BUCKETS_REGION}
```

Comment:
qs.aws.quantumics.bucket.region=${QUANTUMICS_BUCKETS_REGION}
qs.aws.natwest.bucket.region=${NATWEST_BUCKETS_REGION}
Do we need this now?

Reply:
Not required but if we remove this property then getBucketRegions api code also need to be removed.